herbieapp/services/message_publisher/message_publisher.py

```diff
@@ -0,0 +1,38 @@
+import logging
+from herbie import settings
+from herbieapp.models import AbstractBusinessEntity
+from herbieapp.services.message_publisher.google_pub_sub_publisher import GooglePubSubPublisher
+from herbieapp.services.message_publisher.kafka_publisher import KafkaPublisher
+from herbieapp.models.message_models_serializers import EntityUpdateMessage, EntityDeleteMessage
+from herbieapp.services.utils import BusinessEntityUtils
+
+
+class MessagePublisher:
+
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        self._logger = logging.getLogger(__name__)
+        if settings.MESSAGING_PROVIDER == 'kafka':
```

Comment:
It might be a good idea to have a registry class for the resolving the provider.